### ISSUE TITLE
Enhancement: Detect more logging libraries in structured-logging

### ISSUE THREAD

**Issue by pertrai1** [pertrai1/eslint-plugin-llm-core#32]
## Description

The `structured-logging` rule currently detects `console.*` and custom functions like `logError`, `logInfo`. It could be extended to recognize common logging libraries out of the box.

### Pino
```ts
logger.info({ userId }, 'User logged in'); // Good
logger.info(`User ${userId} logged in`); // Bad - flagged
```

### Winston
```ts
winston.info('User logged in', { userId }); // Good
winston.info(`User ${userId} logged in`); // Bad - flagged
```

### Bunyan
```ts
log.info({ userId }, 'User logged in');
```

## Suggested Approach

Add detection for common logger object names (`logger`, `winston`, `bunyan`, `pino`) or make this configurable via options.

```ts
logObjects: ['logger', 'winston', 'bunyan', 'pino', 'log']
```
